CHANGES

```diff
@@ -110,6 +101,11 @@ Notations
 
 Tools
 
+- New warnings system for coqtop/coqc. Can be configured with the flag "-w" or
+  with the Vernacular command "Set Warnings".
+- Warnings are now printed by coqc.
+- In asynchronous mode, Coq is now capable of recovering from errors and
+  continue processing the document.
 - coqc accepts a -o option to specify the output file name
```

Comment:
Well for 8.3, you mentioned the beautifier, -R and compatibility options to coqc/coqtop in the "tools" category :)